src/System.Net.Http.WinHttpHandler/src/Resources/Strings.resx

```diff
@@ -159,4 +159,7 @@
   <data name="net_http_no_concurrent_io_allowed" xml:space="preserve">
     <value>The stream does not support concurrent I/O read or write operations.</value>
   </data>
-</root>
+  <data name="Argument_BufferNotLongEnough" xml:space="preserve">
```

Comment:
Please follow the naming conventions for string ids in this file.

"Argument_BufferNotLongEnough" should be something like "net_http_buffer_Insufficient_length"